ara/ui/templates/partials/host_index_row.html

```diff
@@ -1,51 +1,61 @@
-      {{ host.ok }} OK
+    <a class="badge badge-pill badge-success ara-result-status-badge" title="Search for tasks with status: ok" {% if not static_generation %}href="{% url 'ui:host' host.id %}?status=ok#results"{% endif %}>
+      {{ host.ok }}
     </a>
     {% endif %}
     {% if host.changed %}
-    <a class="badge badge-pill badge-warning ara-result-status-badge" {% if not static_generation %}href="{% url 'ui:host' host.id %}?changed=True#results" title="Search changed results for {{ host.name }}"{% endif %}>
-      {{ host.changed }} CHANGED
+    <a class="badge badge-pill badge-warning ara-result-status-badge" title="Search for tasks with status: changed" {% if not static_generation %}href="{% url 'ui:host' host.id %}?changed=True#results"{% endif %}>
+      {{ host.changed }}
     </a>
     {% endif %}
-    {% if host.failed %}
-    <a class="badge badge-pill badge-danger ara-result-status-badge" {% if not static_generation %}href="{% url 'ui:host' host.id %}?status=failed#results" title="Search failed results for {{ host.name }}"{% endif %}>
-      {{ host.failed }} FAILED
+    {% if host.failed or host.unreachable %}
+    <a class="badge badge-pill badge-danger ara-result-status-badge" title="Search for tasks with status: failed and unreachable" {% if not static_generation %}href="{% url 'ui:host' host.id %}?status=failed&status=unreachable#results"{% endif %}>
+      {{ host.failed |add:host.unreachable }}
     </a>
     {% endif %}
-    {% if host.unreachable %}
-    <a class="badge badge-pill badge-danger ara-result-status-badge" {% if not static_generation %}href="{% url 'ui:host' host.id %}?status=unreachable#results" title="Search unreachable results for {{ host.name }}"{% endif %}>
-      {{ host.unreachable }} UNREACHABLE
+    {% if host.skipped %}
+    <a class="badge badge-pill badge-info ara-result-status-badge" title="Search for tasks with status: skipped" {% if not static_generation %}href="{% url 'ui:host' host.id %}?status=skipped#results"{% endif %}>
+      {{ host.skipped }}
     </a>
     {% endif %}
-    {% if host.skipped %}
-    <a class="badge badge-pill badge-info ara-result-status-badge" {% if not static_generation %}href="{% url 'ui:host' host.id %}?status=skipped#results" title="Search skipped results for {{ host.name }}"{% endif %}>
-      {{ host.skipped }} SKIPPED
+  </td>
+  <td class="text-nowrap">{{ host.updated | format_date }}</td>
+  <td class="text-nowrap">{{ host.playbook.duration | format_duration }}</td>
+  <td>
+    <a {% if not static_generation %}href="{% url 'ui:host_index' %}?name={{ host.name }}&latest=false" title="Search results for {{ host.name }}"{% endif %}>
+      {{ host.name }}
     </a>
+  </td>
+  {% if host.playbook.name is not None %}
+  <td title="Search results for {{ host.playbook.name }}">
+    {% if not static_generation %}
+      <a href="{% url 'ui:host_index' %}?playbook_name={{ host.playbook.name }}">{{ host.playbook.name }}</a>
```

Comment:
```suggestion
      <a href="{% url 'ui:host_index' %}?playbook_name={{ host.playbook.name }}&latest=false">{{ host.playbook.name }}</a>
```

Reply:
added `&latest={{ request.GET.latest | default:'true' }}`. Thus if _Include all playbook reports_ was set before it will be considered in the link as well. 
Does this is fine for you as well?